content/clientlib.config.js

```diff
@@ -28,10 +28,20 @@ module.exports = {
             assets: {
                 js: [
                     "src/scripts/datalayer/v1/polyfill.js",
-                    "node_modules/@adobe/adobe-client-data-layer/dist/adobe-client-data-layer.min.js",
```

Comment:
This would be a breaking change for any customer that relies on `core.wcm.components.commons.datalayer.v1` to bring in the ACDL module.

I think we should instead define a `core.wcm.components.commons.datalayer.v1` clientlib which is lightweight and an `core.wcm.components.commons.datalayer.acdl` clientlib to bring the ACDL module in.